lib/ui/components/cv_header.dart

```diff
@@ -27,7 +28,9 @@ class CVHeader extends StatelessWidget {
                 subtitle,
                 style: Theme.of(context).textTheme.subtitle1.copyWith(
                       fontWeight: FontWeight.bold,
-                      color: Colors.black,
+                      color: ThemeProvider.themeOf(context).id == 'dark'
```

Comment:
I see a lot of conditional statements for switching colors from dark and white mode in multiple files. I believe we should increase reusability by moving them to a theme switching layer which will return the necessary colors in views.